<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsUserTime</class>
 <widget class="QWidget" name="SettingsUserTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Отложенный запуск</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBox_0">
     <property name="suffix">
      <string> сек</string>
     </property>
     <property name="maximum">
      <number>600000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Период опроса РУД</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox_1">
     <property name="suffix">
      <string> мс</string>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Уведомить об окончании режима</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBox_2">
     <property name="suffix">
      <string> мин</string>
     </property>
     <property name="prefix">
      <string>за </string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
